generated/global-autoconfig-properties/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>generated-parent</artifactId>
+        <version>3.5.0-SNAPSHOT</version>
+    </parent>
+
+
+    <artifactId>global-spring-autoconfig-properties</artifactId>
+    <name>Spring Cloud GCP - Global Autoconfig Properties</name>
+    <description>Common properties for generated spring autoconfigurations</description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Does Spring Cloud GCP support Java 8? If yes, we need to make this `1.8`.

Reply:
@zhumin8

Reply:
`spring-cloud-gcp` has unit tests run against Java 8, 11, and 17 - so we probably should align with the parent pom here: 
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/631a1af70a8f22cb2913ffba094284f707540437/pom.xml#L57-L58

Actually, given the parent structures, I wonder if these properties are already visible and inherited by this child pom? (Haven't verified this locally, but if that's the case we can remove this properties block as well.)

Reply:
Yes, I saw it's defined in the root pom, so it should be inherited. Removing!